### PR TITLE
Views

### PR DESCRIPTION
**Pull request by jsug9**
### Hello 👋 
**In this project:**
- Implement the design from the sneak peek wireframes.
- Use methods created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
Captura de Pantalla 2022-07-11 a la(s) 5 11 13 p  m
Captura de Pantalla 2022-07-11 a la(s) 5 11 16 p  m
Captura de Pantalla 2022-07-11 a la(s) 5 11 19 p  m
Captura de Pantalla 2022-07-11 a la(s) 5 11 23 p  m

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,7 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
```
Comment (Davidosky007) [inline app/controllers/users_controller.rb:3]: * Kindly fix this view as given in the wireframe, after creating users through the console i was able to view them 
image
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @jsug9 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking commit messages 👍 

## Required Changes ♻️
* Kindly work on all the views given in the wireframe as it is a requirement for your views to function that way 
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (jsug9) [reply app/controllers/users_controller.rb:3]: Dear @Davidosky007, I think there is a problem on your end. All 4 views are visible in my end:
Captura de Pantalla 2022-07-11 a la(s) 4 54 37 p  m
Captura de Pantalla 2022-07-11 a la(s) 4 54 39 p  m
Captura de Pantalla 2022-07-11 a la(s) 4 54 41 p  m
Captura de Pantalla 2022-07-11 a la(s) 4 54 45 p  m

**Review by mikethreels — APPROVED** [review]
Hi Augusto, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**